### DISCUSSION TITLE
No route found for "POST https://...": Method Not Allowed (Allow: DELETE)

### DISCUSSION THREAD

**Discussion by chyshyya** [symfony/symfony#42740] [Q&A]
**Symfony version(s) affected**: 5.3.6

**Description**
Form with hidden input name _method equals to DELETE failed to submit with Route allowed methods={"DELETE"} 

**How to reproduce**
<form method="post" action="{{ path('universal_item_delete', {'id': item.id}) }}">
    <input type="hidden" name="_method" value="DELETE">
    <input type="hidden" name="_token" value="{{ csrf_token('delete' ~ item.id) }}">
    <button class="btn btn-sm btn-danger" title="Удалить"><i class="fa fa-fw fa-trash-alt"></i></button>
</form>

**Possible Solution**

**Additional context**

**Comment by fd6130** [comment]
If you are using form like this `<form method="POST">`, then you can only submit it through "POST" method but not "DELETE".

**Reply by chyshyya** [reply]
Forms submits only two ways POST by default and GET if presented post="GET"
The problem is that everything works fine but yesterday after integrating code from symfony 5.2 to 5.3.6...
Form with presented _method="DELETE" stops correctly do job

**Comment by chyshyya** [comment]
There is settings in `config/framework.yaml`:
```
framework:
    http_method_override: false
```
By default it's false for now.

**Comment by guillermogfer** [comment]
Why this bug is closed and move into Discussions?

In [current (5.3) documentation](https://symfony.com/doc/current/routing.html#matching-http-methods) it is clearly still saying


```
  HTML forms only support GET and POST methods. If you're calling a route with a different method from an HTML form, add a 
  hidden field called _method with the method to use (e.g. <input type="hidden" name="_method" value="PUT"/>). If you create 
  your forms with Symfony Forms this is done automatically for you.
```
![image](https://user-images.githubusercontent.com/11395867/138323281-9aba8ae3-df1a-42cb-90ca-18a39101cfa4.png)


But it doesn't work.

**Reply by chyshyya** [reply]
Everything is OK.
There is settings in `config/framework.yaml`:
```
framework:
    http_method_override: false
```
By default it's false for now.

**Reply by guillermogfer** [reply]
Thank you so much! Yeah now it's working again.

I missed it in documentation.

**Reply by fd6130** [reply]
I think now you can post the solution as a reply message and then mark it as answer to close the discussion.
